.teamcity/src/main/kotlin/configurations/GitHubMergeQueueCheckPass.kt

```diff
@@ -11,7 +13,8 @@ import model.CIBuildModel
 import model.StageName
 
 class GitHubMergeQueueCheckPass(model: CIBuildModel) : BaseGradleBuildType(init = {
-    id("Check_GitHubMergeQueueCheckPass")
+    id("${model.projectId}_GitHubMergeQueueCheckPass")
+    uuid = "${DslContext.uuidPrefix}_${model.projectId}_GitHubMergeQueueCheckPass"
```

Comment:
❓ I don't think you need to set the uuid separately as long as you set the ID uniquely. What are we doing for our other configurations?

Reply:
Strange. Maybe the `uuid` got set at some point? The id should be relative: https://www.jetbrains.com/help/teamcity/kotlin-dsl-documentation/2023.05/root/project/id.html

Reply:
I think it might be worth to open an issue on Jetbrains Youtrack, or see if there is an existing issue. I don't understand why we don't run into similar problems with our test configurations. Those don't have a UUID, either.